.github/workflows/pull-request.yml

```diff
@@ -39,6 +39,10 @@ jobs:
     name: Run rpc tests
     uses: ./.github/workflows/starknet-rpc-tests.yml
     needs: madara_commands
+  js-tests:
+    name: Run starknet-js compatibility tests
+    uses: petscheit/madara-js-tests@main
```

Comment:
if u are directly using the `petscheit/madara-js-tests@main`, why do we need the `starknet-js-tests.yml`?

Reply:
just saw that u are not even allowed to do that